sickbeard/network_timezones.py

```diff
@@ -44,15 +44,15 @@ def update_network_dict():
     """Update timezone information from Medusa repositories"""
 
     url = 'https://cdn.pymedusa.com/sb_network_timezones/network_timezones.txt'
-    url_data = helpers.getURL(url, session=helpers.make_session(), returns='response')
-    if not url_data.text:
+    url_data = helpers.getURL(url, session=helpers.make_session(), returns='text')
```

Comment:
This is not the correct way to return text with requests. See documentation: http://docs.python-requests.org/en/master/user/quickstart/#response-content